packages/nimble-components/src/table/specs/table-column-placeholder-hld.md

```diff
@@ -0,0 +1,151 @@
+
+There are two places where a value is displayed in the table that placeholders need to be considered: (1) in a cell and (2) in a group row. The cell placeholder will be configurable on each column that supports having a placeholder, and it will default to an empty string. The group row placeholder will come from the table's localization provider, and it will not be configurable through a column's API.
+
+Placeholders within a cell will be rendered with nimble's placeholder font, which is currently 60% opacity of nimble's body font.
+
+Placeholders within a group row will have no special visual treatment.
+
+Below is an example of what placeholders will look like in the table. In this example, the table is grouped by the "Quote" column, which has been configured to have a placeholder value of "None".
+
+![Placeholder text example](./spec-images/PlacholderText.png)
+
+### Column-specific decisions
+
+The exact behavior of placeholders in each existing table column is described below.
+
+#### Text column
+
+| Special-cased field values | Cell display                                                 | Group row display |
+| -------------------------- | ------------------------------------------------------------ | ----------------- |
+| `undefined`                | column placeholder, or empty if no placeholder is configured | `"No value"`      |
+| `null`                     | column placeholder, or empty if no placeholder is configured | `"No value"`      |
+| `''`                       | \<empty cell>                                                | `"Empty"`         |
+
+#### Anchor column
+
+| Special-cased field values                          | Cell display                                                       | Group row display |
+| --------------------------------------------------- | ------------------------------------------------------------------ | ----------------- |
+| Both label and href are `undefined` or `null`       | column placeholder, or empty if no placeholder is configured       | `"No alias"`      |
+| Label is `undefined` or `null` with defined href    | href value is used as the link's href and the link's display value | `"No alias"`      |
+| Label is defined with href of `undefined` or `null` | label as a plain string with no link                               | The label         |
+| Label is `''` with any href                         | \<empty cell>                                                      | `"Empty"`         |
+
+#### Number column
+
+| Special-cased field values | Cell display                                                 | Group row display |
+| -------------------------- | ------------------------------------------------------------ | ----------------- |
+| `undefined`                | column placeholder, or empty if no placeholder is configured | `"No value"`      |
+| `null`                     | column placeholder, or empty if no placeholder is configured | `"No value"`      |
+
```

Comment:
I've added some notes, but I wasn't entirely sure what to say about the special cases. Did you have anything in mind?

I also, didn't list that -0 and +0 shouldn't be used because I wasn't aware of any problems they would cause -- although -0, +0, and 0 all get treated identically, so maybe that's the note to make about those values?